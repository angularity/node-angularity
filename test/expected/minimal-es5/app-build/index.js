(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
"use strict";var r=require("___________________0");var e=require("_______________________1");angular.module("browser-image-diff-client",["ui.router"]).controller("AppController",e).config(r);
},{"_______________________1":3,"___________________0":4}],2:[function(require,module,exports){
module.exports="<h1>Minimal {{title}}</h1>";
},{}],3:[function(require,module,exports){
"use strict";module.exports=function(t){t.title="Hello World"};
},{}],4:[function(require,module,exports){
"use strict";module.exports=function(e,t){t.otherwise("/");e.state("home",{url:"/",template:require("_________2"),controller:"AppController"})};
},{"_________2":2}]},{},[1])


//# sourceMappingURL=index.js.map