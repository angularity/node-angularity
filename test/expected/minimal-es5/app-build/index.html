<!DOCTYPE html>
<html ng-app="browser-image-diff-client">
<head lang="en">
    <!-- bower:css -->
    <!-- endinject -->
    <!-- inject:css -->
    <link rel="stylesheet" href="/app-build/index.css">
    <!-- endinject -->
</head>
<body>
    <ui-view></ui-view>
    <!-- bower:js -->
    <script src="/bower_components/jquery/dist/jquery.js"></script>
    <script src="/bower_components/angular/angular.js"></script>
    <script src="/bower_components/angular-ui-router/release/angular-ui-router.js"></script>
    <script src="/bower_components/angular-mocks/angular-mocks.js"></script>
    <!-- endinject -->
    <!-- inject:js -->
    <script src="/app-build/index.js"></script>
    <!-- endinject -->
</body>
</html>